util/aws/http_client.cc

```diff
@@ -94,8 +94,9 @@ std::shared_ptr<Aws::Http::HttpResponse> HttpClient::MakeRequest(
     buf_body = dynamic_cast<boost::interprocess::bufferstream*>(request->GetContentBody().get());
     // Only copy if we don't know the type.
     if (!buf_body) {
-      int content_size;
-      absl::SimpleAtoi(request->GetContentLength(), &content_size);
+      unsigned content_size;
+      CHECK(absl::SimpleAtoi(request->GetContentLength(), &content_size));
```

Comment:
Are you sure you want to crash if the request is invalid?

Reply:
I do not know. This function does not return an error code and the previous code was UB.

Reply:
:shrug: